source/text/implementation/vss-string_vectors.adb

```diff
@@ -98,6 +98,25 @@ package body VSS.String_Vectors is
       VSS.Implementation.String_Vectors.Unreference (Self.Data);
    end Clear;
 
+   --------------
+   -- Contains --
+   --------------
+
+   function Contains
+     (Self : Virtual_String_Vector'Class;
+      Item : VSS.Strings.Virtual_String'Class) return Boolean
+   is
+      use VSS.Strings;
+   begin
+      for Element of Self loop
```

Comment:
This is very inefficient from performance point of view:
 - hidden cursor operations
 - return of object of controlled type

It is better to lower level of implementation and move it into VSS.Implementations.String_Vectors.